deploy.pl

```diff
@@ -36,15 +37,33 @@
 
 my $ua = Mojo::UserAgent->new;
 
+my $application_id = $marathon_json->{id};
+
 my $app_url = URI->new("$marathon_apps_url/".uri_escape($marathon_json->{id}))->canonical->as_string();
+
 my $res = $ua->put($app_url => json => $marathon_json)->res();
 
 if ($res->code != 200 && $res->code != 201) {
     die $res->to_string();
 }
 
```

Comment:
what is $option? :-)

And should it be opt-in or opt-out?

I would prefer opt-out...